packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -159,15 +156,10 @@ describeCompat("GC data store tombstone tests", "NoCompat", (getTestObjectProvid
 			await loadSummarizer(container);
 		const summaryVersion = (await summarize(summarizer1)).summaryVersion;
 
-		// If we added a DDS above, this trailing op will trigger a codepath that over-eagerly detects that
-		// DDS reference as if it's new, and wipes out the unreference tracking for the datastore.
-		// So we must skip the trailing op to avoid that case.
-		if (!includeDds) {
-			// TODO: trailing op test - note because of the way gc is currently structured, the error isn't logged,
-			// but it is detected - it's stored in the pending queue and the container closes before the error is sent.
-			testDataObject._root.set("send while unreferenced", "op");
-			await provider.ensureSynchronized();
-		}
+		// TODO: trailing op test - note because of the way gc is currently structured, the error isn't logged,
```

Comment:
We have trailing op tests now so this comment can be removed - https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-end-to-end-tests/src/test/gc/gcTrailingOps.spec.ts.